src/raftstore/store/util.rs

```diff
@@ -764,6 +765,24 @@ impl Engines {
     }
 }
 
+pub fn format_split_info(region_id: u64, split_keys: &[Vec<u8>]) -> String {
+    if split_keys.len() == 1 {
+        format!(
+            "{} with key {:?}",
+            region_id,
+            split_keys.first().as_ref().map(|k| escape(k)),
+        )
+    } else {
+        format!(
+            "{} with {} keys range from {:?} to {:?}",
```

Comment:
`escape` displays the content in debug format already as it *escapes* the invisible chars. Better use `"{}"`, note that `"` is necessary as `escape` doesn't add it for you.